### ISSUE TITLE
Question about documentation on Context

### ISSUE THREAD

**Issue by Th0masL** [OpenNebula/terraform-provider-opennebula#172]
On the [virtual machine documentation](https://github.com/OpenNebula/terraform-provider-opennebula/blob/master/website/docs/r/virtual_machine.html.markdown#example-usage), it is written that the context has to be defined as `context {}`, but on my side, I can only get it work with `context = {}`.

From documentation (doesn't work in production)
```
resource "opennebula_virtual_machine" "haproxy" {
    context {
        NETWORK      = "YES"
        HOSTNAME     = "$NAME"
        START_SCRIPT ="yum upgrade"
    }
    <...>
}
```

What I have in production (that works)
```
resource "opennebula_virtual_machine" "haproxy" {
    context = {
        NETWORK      = "YES"
        HOSTNAME     = "$NAME"
        START_SCRIPT ="yum upgrade"
    }
    <...>
}
```

Which format is the correct one ?

**Comment by treywelsh** [comment]
It's an old problem that we didn't fix until now: #56 
The documentation fix shouldn't be painful, do you want to give it a try ?

**Comment by Th0masL** [comment]
Sure, here is the PR https://github.com/OpenNebula/terraform-provider-opennebula/pull/173
